README.md

```diff
@@ -82,21 +82,19 @@ These instructions will get Snow Owl up and running on your local machine for de
 ## Prerequisites
 
 Snow Owl is an Equinox-OSGi based server. To develop plug-ins for Snow Owl you need to use Eclipse as IDE: 
-* Download Eclipse IDE for Eclipse Committers 2021-12 package from here: https://www.eclipse.org/downloads/packages/release/2021-12/r/eclipse-ide-eclipse-committers
+* Download Eclipse IDE for Eclipse Committers 2023-09 package from here: https://www.eclipse.org/downloads/packages/release/2023-12/r/eclipse-ide-eclipse-committers
```

Comment:
Now it's 2023-09 and 2023-12 at the same time.

Reply:
Fixed